pygameMenu/scrollarea.py

```diff
@@ -297,6 +297,20 @@ def get_view_rect(self):
 
         return rect
 
+    def get_actual_scrollbar_thickness(self, orientation):

```

Comment:
naming `get_scrollbar_thickness` is enough , no?

Reply:
that's better